Technical-Steering-Committee/release.md

```diff
@@ -0,0 +1,79 @@
+3. RC is built and tested with automatic test suite
+4. RC is deployed and tested by squads. 
+5. RC is approved by the TSC vote
+6. Release is published
+7. Documentation is published
+8. Release retrospective is held. 
+
+
+## Milestones and Release Candidates
+
+Zowe Milestones are time-based releases with a schedule published on: [https://github.com/zowe/community/blob/master/Project%20Management/Schedule/Zowe%20PI%20%26%20Sprint%20Cadence.md#sprint-cadence](https://github.com/zowe/community/blob/master/Project%20Management/Schedule/Zowe%20PI%20%26%20Sprint%20Cadence.md#sprint-cadence). The schedule also contains dates for Code Freeze, RC builds, System demo and the expected publishing. 
+
+Each Squad will decide if there will be a deliverable depending on the content that went into the Release. Squads are self-governing and are free to have their own schedules as long as they meet the requirements of the release as defined by the community.
+
+
+## Release Approval Procedure
+
+
+### Process when potential stop-ship was found
+
+The TSC via zowe-tsc channel is informed that the potential stop-ship bug was found. The TSC then based on the recommendation from the affected squad/s vote on whether the found bug is a stop ship. The available options are: wait for the fix, return to the previous version of the component, agree it’s not a stop ship. The TSC votes in the channel and the process follows based on the found decision. 
+
+We expect the squads to be able to revert back to the previous version in case it isn’t possible to fix the issue in a time frame to be able to produce release on time. 
+
+
+### Approval of release passing the tests
+
+The TSC is informed by the Release manager that a new release is ready for public distribution. The appointed member of TSC will review the release and ensure that all requirements that are needed for legal and technical release are met. These include, but are not limited to, updated Documentation, correct LICENSE files and code scans showing no known vulnerabilities.
+
+The vote is going to happen in the zowe-tsc channel as explained in the TSC Structure and TSC Charter. The option is to vote for releasing or for stopping the release. When the votes approves the release a TSC member, or the chair, will notify the release manager. The Release manager will then make the artifacts available and notify the OMP Director of the new release to track the LLC release of a new set of binaries.
+
+
+## Release Numbering
+
+Each release will be identified by a version number. These numbers are used according to a specific scheme that will give you additional information about the release.  The version numbers are of the form x.y.z-[GA | beta | yyyymmdd] (Semantic Versioning) or major.minor.micro. The final designation indicates whether this is the official Generally Available (GA) version, a beta version or an interim build. Only releases that end with **-GA** are considered official releases.
```

Comment:
`Only releases that end with **-GA** are considered official releases.` - In practice, we announce GA version without any suffix.